src/app/shared/entry/info-tab-checker-workflow-path/info-tab-checker-workflow-path.component.html

```diff
@@ -1,4 +1,4 @@
-<li *ngIf="((checkerId$ | async) || (canAdd$ | async)) && (isRefreshing$ | async) === false">
+<li *ngIf="(checkerId$ | async) || (canAdd$ | async)">
```

Comment:
The `Add` button on line 19 should also have ` && (isRefreshing$ | async) === false` in its `ngIf` condition so that users can't click it while the page is refreshing.